Tests/CustomEntitySearchTest/CustomEntitySearchTest.cs

```diff
@@ -0,0 +1,167 @@
+            var response = client.PostAsync(TestData.hostAddress, jsonContent).Result;
+            string responseString = response.Content.ReadAsStringAsync().Result;
+            WebApiResponseRecord output = new WebApiResponseRecord();
+            try
+            {
+                output = JsonConvert.DeserializeObject<WebApiResponseRecord>(responseString);
+            }
+            catch
+            {
+                Assert.Fail("Skill failed to handle an empty test. Errored out.");
+            }
+            string checkTest6 = TestData.GetOutput(TestData.largeWordsQuickResultInputWords, TestData.largeWordsQuickResultExpectedOutput);
+            Assert.AreEqual(checkTest6, responseString, false);
+        }
+
+        [TestMethod]
+        public void LargeDatasetQuickResult()
+        {
+            //tests against a large number of documents inputted (loadtest)
+            string content = TestData.GetPayload(TestData.largestText, TestData.largeTextQuickResultInputWords, TestData.numDocs);
+            HttpContent jsonContent = new StringContent(content, null, "application/json");
+            var response = client.PostAsync(TestData.hostAddress, jsonContent).Result;
+            string responseString = response.Content.ReadAsStringAsync().Result;
+            WebApiResponseRecord output = new WebApiResponseRecord();
+            try
+            {
+                output = JsonConvert.DeserializeObject<WebApiResponseRecord>(responseString);
+            }
+            catch
+            {
+                Assert.Fail("Skill failed to handle an empty test. Errored out.");
+            }
+            string checkTest7 = TestData.GetOutput(TestData.largeTextQuickResultInputWords, TestData.largeTextQuickResultExpectedOutput, TestData.numDocs);
+            Assert.AreEqual(checkTest7, responseString, false);
+        }
+
+        [TestMethod]
+        public void LargeNumWordsQuickResult()
+        {
+            // tests against a large number of patterns in words array
+            string largeNumWords = TestData.GetPayload(TestData.largestText, TestData.largestWords);
+            HttpContent jsonContent = new StringContent(largeNumWords, null, "application/json");
+            var response = client.PostAsync(TestData.hostAddress, jsonContent).Result;
+            string responseString = response.Content.ReadAsStringAsync().Result;
+            WebApiResponseRecord output = new WebApiResponseRecord();
+            try
+            {
+                output = JsonConvert.DeserializeObject<WebApiResponseRecord>(responseString);
+            }
+            catch
+            {
+                Assert.Fail("Skill failed to handle an empty test. Errored out.");
+            }
+            string checkTest8 = TestData.GetOutput(TestData.largestWords, TestData.largeNumWordsQuickResultExpectedOutput);
+            Assert.AreEqual(checkTest8, responseString, false);
+        } 
+    }
```

Comment:
Would be useful to write more tests around word delineation. For example, it looks like the pattern "car" would currently match a document containing "We'll perform a cardiovascular operation at noon."

Reply:
Yes, it would. I changed it to be exact pattern matching for now.